tests/unit/components/sl-date-time-test.js

```diff
@@ -16,15 +16,28 @@ test( 'Default class names are present', function( assert ) {
 
 test( 'Attribute "datetime" is properly set', function( assert ) {
     this.subject({
-        timezone: 'America/Chicago',
-        value: new Date( 2015, 0, 1 )
+        format: 'datetime',
+        timezone: 'America/Denver',
```

Comment:
I'm assuming that this change is related to your comment on the PR about how the test was incorrect because of the timezone used and that it was not failing in your dev environment?  If so, this is not an acceptable fix.  All this change does is "fix" it for any devs in the `America/Chicago` timezone but has moved the problem to those in the `America/Denver` timezone.  The approach of this test needs to be re-visited.